CMakeLists.txt

```diff
@@ -93,8 +93,12 @@ if(NOT DEFINED ROOT_CXX_STANDARD)
     set(ROOT_CXX_STANDARD 17)
   elseif("cxx_std_20" IN_LIST ROOT_COMPILE_FEATURES)
     set(ROOT_CXX_STANDARD 20)
+  elseif("cxx_std_14" IN_LIST ROOT_COMPILE_FEATURES)
+    set(ROOT_CXX_STANDARD 14)
+  elseif("cxx_std_11" IN_LIST ROOT_COMPILE_FEATURES)
+    set(ROOT_CXX_STANDARD 11)
```

Comment:
Is there a reason not to put these in order? 11, 14, 17, 20, else

Reply:
Or simply a loop
```
foreach(std IN ITEMS 20 17 14 11)
  if("cxx_std_${std}" IN_LIST ROOT_COMPILE_FEATURES)
    set(ROOT_CXX_STANDARD ${std})
    break()
  endif()
endforeach()
```

Reply:
It's a loop now.